<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormFeedDetails</class>
 <widget class="QDialog" name="FormFeedDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="m_tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabUpdating">
      <attribute name="title">
       <string>Auto-updating</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Auto-update</string>
         </property>
         <property name="buddy">
          <cstring>m_cmbAutoUpdateType</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QComboBox" name="m_cmbAutoUpdateType">
           <property name="toolTip">
            <string>Select the auto-update strategy for this feed. Default auto-update strategy means that the feed will be update in time intervals set in application settings.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="TimeSpinBox" name="m_spinAutoUpdateInterval">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabNetwork">
      <attribute name="title">
       <string>Network</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QGroupBox" name="m_gbAuthentication">
         <property name="toolTip">
          <string>Some feeds require authentication, including GMail feeds. BASIC, NTLM-2 and DIGEST-MD5 authentication schemes are supported.</string>
         </property>
         <property name="title">
          <string>Requires HTTP authentication</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Username</string>
            </property>
            <property name="buddy">
             <cstring>m_txtUsername</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="LineEditWithStatus" name="m_txtUsername" native="true"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Password</string>
            </property>
            <property name="buddy">
             <cstring>m_txtPassword</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="LineEditWithStatus" name="m_txtPassword" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="m_buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEditWithStatus</class>
   <extends>QWidget</extends>
   <header>lineeditwithstatus.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TimeSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>timespinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FormFeedDetails</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>342</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
